### PR TITLE
Prepare 6.0.0

### LINKED ISSUE #308: Drop IE10
https://github.com/Level/abstract-leveldown/blob/e7285bb66fe802d32715feb2e612a219e7ddba55/.airtap.yml#L5-L7

### LINKED ISSUE #311: Update changelog for v6
It's missing some items; and others (related to the refactoring of the test suite) can be merged.

### PR DESCRIPTION
**Pull request by vweevers**
Closes #308, closes #311.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+| **Lars-Magnus Skog**       | [**@ralphtheninja**](https://github.com/ralphtheninja)               | [**@ralph@social.weho.st**](https://social.weho.st/@ralph)               |
+| **Rod Vagg**               | [**@rvagg**](https://github.com/rvagg)                               | [**@rvagg@twitter**](https://twitter.com/rvagg)                          |
+| **Vincent Weevers**        | [**@vweevers**](https://github.com/vweevers)                         | [**@vweevers@twitter**](https://twitter.com/vweevers)                    |
+| **Julian Gruber**          | [**@juliangruber**](https://github.com/juliangruber)                 | [**@juliangruber@twitter**](https://twitter.com/juliangruber)            |
+| **David Björklund**        | [**@kesla**](https://github.com/kesla)                               | [**@david_bjorklund@twitter**](https://twitter.com/david_bjorklund)      |
+| **ralphtheninja**          | [**@308049+ralphtheninja**](https://github.com/308049+ralphtheninja) |                                                                          |
```
Comment (vweevers) [inline CONTRIBUTORS.md]: Hmm, what's this

Comment (ralphtheninja) [reply CONTRIBUTORS.md]: That looks odd indeed. Maybe a change directly on github?

Comment (vweevers) [reply CONTRIBUTORS.md]: It's mostly from merge commits:

image

I'll see if I can detect this case in `remark-git-contributors`. I.e. strip `@users.noreply.github.com` and the number prefix (no idea what that is) to get a username.

Comment (vweevers) [reply CONTRIBUTORS.md]: In the mean time, the rest is ready for review

**Comment by vweevers** [comment]
Mandatory gif

done-downsized

**Comment by ralphtheninja** [comment]
Hahaha I just love that gif :smile:

**Pull request merged**